recipes/eqt/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "eqt" %}
+{% set version = "0.7.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/eqt-{{ version }}.tar.gz
+  sha256: 8cb506756b7bbc9fbd3da5b020bd9c3b41335b2e716d3512e06c6cab7653ebbb
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - setuptools >=42
+    - setuptools-scm >=3.4
+    - pip
+  run:
+    - python >=3.7
+    - pyside2
+    - qdarkstyle
+
+test:
+  imports:
+    - eqt
+  commands:
+    - pip check
+    - env CI=1 pytest  # [not linux]
```

Comment:
Are you planning on running the full test suit tests here? If not, you don't need this, the pytest dependencies below, and the source_files.

Reply:
not really needed as all tests are run upstream anyway. Removed :)